dvc/remote/azure.py

```diff
@@ -119,15 +119,15 @@ def _list_paths(self, bucket, prefix):
     def list_cache_paths(self):
         return self._list_paths(self.path_info.bucket, self.path_info.path)
 
-    def _upload(
+    def upload_file(
```

Comment:
No, this is not `upload_file()`. Now `upload()` is upload file and this is `upload_file_no_checks_no_progress_finish_target_no_error_handling` or `_low_level_upload` or just `_upload`

Reply:
Or `do_upload`, what do you think?

Reply:
1. It should stay private.
2. What's wrong with `_upload()`?

Reply:
I need to use _download in dvc/repo/get_url.py, which would make me use it as a private method. I'll re-check in the morning.